src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <p className="Person__name">
+      {`My name is ${person.name}`}
+    </p>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )
+    }
+    {person.isMarried ? (
+      <>
+        {person.sex === 'm' ? (
+          <p className="Person__partner">
+            {`${person.partnerName} is my wife`}
+          </p>
+        ) : (
+          <p className="Person__partner">
+            {`${person.partnerName} is my husband`}
+          </p>
+        )}
+      </>
+    ) : (
+      <>
+        <p className="Person__partner">
+          I am not married
+        </p>
+      </>
+    )
+    }
```

Comment:
avoid nested ternary. use a variable instead.
`const partner = person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband``